### PR TITLE
Editor: Display placeholder on empty content

### PR DESCRIPTION
**Pull request by aduth**
This pull request seeks to add a placeholder label to the editor content area when the post has no contents.

Before|After
---|---
After|Before

__Testing instructions:__

Confirm that in Visual mode the placeholder label is only shown when the post editor has no content and when it is not focused. In Edit mode, the placeholder should be shown until some characters are entered.

Particularly, try a variety of situations such as loading a draft while another is opened, or starting a new draft while a post is currently visible.

1. Navigate to the [Calypso post editor](http://calypso.localhost:3000/post)
2. Select a site, if prompted
3. Assuming the default tab is Visual, note that a placeholder label is shown
4. Focus the editor area
5. Note that the placeholder disappears
6. (Optional) Enter some post content
7. Click elsewhere on the page to unfocus the editor
8. Note that the placeholder label is only shown if no content was added in Step 6

### CODE REVIEW

File: client/components/tinymce/plugins/placeholder/style.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+body.is-showing-placeholder:not( :focus )::before {
+	font-style: italic;
```
Comment (mtias) [inline client/components/tinymce/plugins/placeholder/style.scss]: I think we shouldn't make this italics.

**Comment by aduth** [comment]
@mtias : Indeed, looks better without the italics:

image

(Also lowered the opacity a bit more so that it's not confused with content)

**Comment by mtias** [comment]
Perfect!

**Comment by aduth** [comment]
Was not the reference, but finding that Medium uses a very similar placeholder:

medium placeholder

Should we revise the text or disregard the similarities?

**Comment by kellychoffman** [comment]
+1 to non italics. 

Looks good, but I don't think its great that its so similar to Medium. 

Do you think we should have the placeholder the same for both posts and pages? I'm not sure the current "Tell me your story..." makes sense for pages. 

Open to other ideas, but what about: `Write your post...` and `Create your page...`

**Comment by rickybanister** [comment]
Something like `Post body...` and `Page body...` are a little bland, but they are nouns the way `Title` is—might be worth keeping them similar...

**Comment by kellychoffman** [comment]
I'm not sure they need to be similar.

**Comment by aduth** [comment]
Some ideas exchanged through conversation with our editorial friends:

- Get started!
- Start typing!
- Your words go here
- What do you have to say?
- What’s on your mind?

/cc @michelleweber, @benhuberman, @mikedang

**Comment by designsimply** [comment]
Adding one more for consideration:

* Start writing here…

**Comment by kellychoffman** [comment]
> Get started!
> Start typing!

Adding an exclamation mark sounds a bit like we are shouting. Do this! Now!

> Your words go here

A little bland, sounds like a computer instead of a human. 

> What do you have to say?

Doesn't inspire confidence. "Hmm, maybe I don't have anything important to say..." 

> What’s on your mind?

I like this for posts. We just need a version of this for Pages.

**Comment by aduth** [comment]
There are a few outstanding bugs with this pull request, and with the layout revisions introduced in #2466, the need for a placeholder may be moot, as the updated edges of the toolbar more clearly illustrate the boundaries of the editor. We can reconsider this again in the future if user confusion is reported.

**Comment by kellychoffman** [comment]
:+1: Good call.

**Pull request closed without merging**